Index: _includes/header.cfm
===================================================================
--- _includes/header.cfm	(revision 3733)
+++ _includes/header.cfm	(working copy)
@@ -74,7 +74,69 @@
     </cfif>
 </head>
 
+
+
+<!---NDF: readonly, 2014-01-31. --->
+<style>
+body{ color: #222; background: #fcfcfc; }
+
+.readonly-msg {
+  position:absolute; left: 100px; top: 22px; padding: 5px; border: 3px solid #900;
+  background: #f7f7f7; font-size: 1em; text-align: center;
+}
+/* edit and Edit, create and Create... */
+.is-readonly a[href *= "dit"], .is-readonly a[href *= "reate"], .is-readonly a[href *= "ubscribe"],
+.is-readonly a[href *= "hange"], .is-readonly [href *= "elete"] {
+  /*display: none;*/
+  color: #777;
+  text-decoration: line-through;
+}
+.is-readonly a[href *= "dit"]:after, .is-readonly [href *= "reate"]:after, .is-readonly [href *= "ubscribe"]:after,
+.is-readonly [href *= "hange"]:after, .is-readonly [href *= "elete"]:after {
+  content: " (Disabled)";
+  text-decoration: none;
+  color: #900;
+}
+.readonly-no-access {
+  text-align: center; margin: 3em;
+}
+.readonly-no-access img { margin: 2em; }
+</style>
+<cfif IsDefined("REQUEST.is_readonly") and REQUEST.is_readonly AND
+  (REQUEST.readonly_user eq CLIENT.iamid OR IsDefined("URL.readonly")) >
+
+  <cfif IsDefined("URL.method") and
+    (URL.method contains "edit" OR URL.method contains "create" OR URL.method CONTAINS "subscribe"
+    OR URL.method CONTAINS "add" OR URL.method CONTAINS "delete") >
+
+    <!---<cferror template = "../CustomTags/sitewide/ErrorResourceNotFound.cfm" type = "exception">--->
+
+<body class="readonly-no-access">
+
+    <h1>This page is no longer accessible</h1>
+    <p><a href="javascript:window.history.back();">Go back to the previous page</a>.
+    <p><a href="#TODO" onclick="alert('Under construction.')">Find out more</a>.
+    <p><a href="/public/"><img src="/images/title_knowledgenetwork.gif" alt="OU Knowledge Network" title="Open University Knowledge Network" /></a>
+    <p><small>[ Method: <cfoutput>#URL.method#</cfoutput>; Iamid: <cfoutput>#Client.iamid#</cfoutput> ]
+
+    <cfabort /><!---showError = "Custom error..." />--->
+  </cfif>
+
+<body class="is-readonly">
+
+  <p class=readonly-msg >From 14 February 2014, the Knowledge Network will be read-only.
+    <br><a href="#TODO" onclick="alert('Under construction.')">Find out more</a>.
+  </p>
+
+<cfelse>
+
 <body>
+
+</cfif>
+<!---NDF: END.--->
+
+
+
 	<div id="scroll"></div>
 	<!--- draw the OU header bar --->
 	<cfinclude template="/includes/header-full-noresize.html">
Index: server_defaults.cfm
===================================================================
--- server_defaults.cfm	(revision 3733)
+++ server_defaults.cfm	(working copy)
@@ -2,7 +2,14 @@
 		*** development server *** in banner bar)--->
 
 
+<!---NDF: readonly, 2014-01-31. --->
+<cfset REQUEST.is_readonly = "true">
+<cfset REQUEST.readonly_user__YY = "0">
+<cfset REQUEST.readonly_user = "14298">
 
+
+
+
 <cfset request.contact_email_address = "knowledge-network@open.ac.uk">
 <cfset request.contact_email_name = "Knowledge Network team">
 <!--- E-mail address for error messages --->
