Index: _includes/header.cfm
===================================================================
--- _includes/header.cfm	(revision 3733)
+++ _includes/header.cfm	(working copy)
@@ -74,7 +74,146 @@
     </cfif>
 </head>
 
+
+
+
+<!---NDF: readonly, 2014-01-31. --->
+<cfif IsDefined("REQUEST.is_readonly") and REQUEST.is_readonly >
+<!---<cfif IsDefined("REQUEST.is_readonly") and REQUEST.is_readonly AND
+  (REQUEST.readonly_user eq CLIENT.iamid OR IsDefined("URL.readonly")) >--->
+<style>
+body{ color: #222; background: #fcfcfc; }
+a#ou-skip2{ top: 15px; }
+
+.readonly-msg {
+  position:absolute; left: 100px; top: 22px; padding: 3px; border: 1px solid #900;
+  background: #fafafa; font-size: .85em; text-align: center;
+}
+/* Case-insensitive: "edit" and "Edit", create and Create... */
+.is-readonly a[href *= "dit"], .is-readonly a[href *= "reate"], .is-readonly a[href *= "ubscribe"],
+.is-readonly a[href *= "hange"], .is-readonly [href *= "elete"] {
+  /*display: none;*/
+  color: #777;
+  text-decoration: line-through;
+}
+.is-readonly a[href *= "dit"]:after, .is-readonly [href *= "reate"]:after, .is-readonly [href *= "ubscribe"]:after,
+.is-readonly [href *= "hange"]:after, .is-readonly [href *= "elete"]:after {
+  content: " (Disabled)";
+  text-decoration: none;
+  color: #900;
+}
+.readonly-no-access {
+  text-align: center; margin: 3em;
+}
+.readonly-no-access img { margin: 2em; }
+</style>
+<style>
+#simplemodal-container a.modalCloseImg {
+	background:url(http://simplemodal.googlecode.com/svn/tags/1.3/test/img/x.png) no-repeat; /* adjust url as required */
+	width:25px;
+	height:29px;
+	display:inline;
+	z-index:3200;
+	position:absolute;
+	top:-15px;
+	right:-18px;
+	cursor:pointer;
+}
+#simplemodal-container .readonly-msg {
+  line-height: 1.7em;
+  font-size: 1.1em;
+  padding: 8px;
+  border-width: 3px;
+  border-radius: 4px;
+}
+</style>
+
+  <cfif (DateCompare(Now(), REQUEST.readonly_date) eq 1  OR  IsDefined("URL.readonly"))
+    AND
+        IsDefined("URL.method")
+    AND
+    (URL.method contains "edit" OR URL.method contains "create" OR URL.method CONTAINS "subscribe"
+    OR URL.method CONTAINS "add" OR URL.method CONTAINS "delete") >
+
+    <!---<cferror template = "../CustomTags/sitewide/ErrorResourceNotFound.cfm" type = "exception">--->
+
+<body class="readonly-no-access">
+
+    <h1>This page is no longer accessible</h1>
+    <p>The Knowledge Network is now in read-only mode.
+    <p><a href="javascript:window.history.back();">Go back to the previous page</a>.
+    <p><a href="#TODO" onclick="alert('Under construction.')">Find out more</a>.
+    <p><a href="/public/"><img src="/images/title_knowledgenetwork.gif" alt="OU Knowledge Network" title="Open University Knowledge Network" /></a>
+    <p><small>[ Method: <cfoutput>#URL.method#</cfoutput>; Iamid: <cfoutput>#Client.iamid#</cfoutput> ]
+
+    <cfabort /><!---showError = "Custom error..." />--->
+  </cfif>
+
+<body class="is-readonly">
+
+  <cfif DateCompare(Now(), REQUEST.readonly_msg_date) eq 1 OR IsDefined("URL.readonly") >
+
+  <p class=readonly-msg id=readonly-msg >From <cfoutput>#DateFormat(REQUEST.readonly_date, "d mmmm yyyy")#</cfoutput>,
+    the Knowledge Network will be read-only.
+    <br><a href="#TODO" onclick="alert('Under construction.')">Find out more</a>.
+    <!---<a href="#close" title="Close">x</a>--->
+  </p>
+
+  <cfif REQUEST.readonly_modal >
+  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+  <script src="//cdn.jsdelivr.net/simplemodal/1.4.4/jquery.simplemodal.min.js"></script>
+  <script>
+  (function () {
+
+    if (document.cookie.match(/kn_readonly=1/)) {
+      return;
+    }
+
+    // http://ericmmartin.com/projects/simplemodal/
+    $("#readonly-msg").modal({
+      opacity: 70,
+      overlayCss: {
+        background: "#222"
+      },
+      overlayClose: true,
+      onClose: function (dlg) {
+        createCookie("kn_readonly", 1);  // Session cookie?
+
+        $.modal.close();
+      }
+    });  //$("#readonly-msg").hide();
+    $(".simplemodal-close").attr({
+      tabindex: 0,
+      role: "button"
+    });
+
+
+    // http://quirksmode.org/js/cookies.html
+    function createCookie(name,value,days) {
+      if (days) {
+        var date = new Date();
+        date.setTime(date.getTime()+(days*24*60*60*1000));
+        var expires = "; expires="+date.toGMTString();
+      }
+      else var expires = "";
+      document.cookie = name+"="+value+expires+"; path=/";
+    }
+
+  })();
+  </script>
+  </cfif>
+  </cfif>
+
+<cfelse>
+
 <body>
+
+</cfif>
+<!---NDF: END.--->
+
+
+
+
 	<div id="scroll"></div>
 	<!--- draw the OU header bar --->
 	<cfinclude template="/includes/header-full-noresize.html">
Index: server_defaults.cfm
===================================================================
--- server_defaults.cfm	(revision 3733)
+++ server_defaults.cfm	(working copy)
@@ -2,7 +2,17 @@
 		*** development server *** in banner bar)--->
 
 
+<!---NDF: readonly, 2014-01-31. --->
+<cfset REQUEST.is_readonly = "true">
+<cfset REQUEST.readonly_user__YY = "0">
+<cfset REQUEST.readonly_user = "14298">
+<cfset REQUEST.readonly_date = CreateDate(2014, 03, 14) /><!--- 27 Feb, 14 Mar --->
+<cfset REQUEST.readonly_msg_date = CreateDate(2014, 03, 03) /><!--- 21 Feb, 3 Mar --->
+<cfset REQUEST.readonly_modal = "true" />
 
+
+
+
 <cfset request.contact_email_address = "knowledge-network@open.ac.uk">
 <cfset request.contact_email_name = "Knowledge Network team">
 <!--- E-mail address for error messages --->
